### PR TITLE
fix issues of pipeline-service test

### PR DESCRIPTION
**Pull request by xinredhat**
This PR is to fix issue of  [PLNSRVCE-1492](https://issues.redhat.com/browse/PLNSRVCE-1492)

### CODE REVIEW

**Comment by xinredhat** [comment]
this PR is ready for review. I have already tested that if the pipeline-service tests  fail, the `test-pipeline-service` task will also fail.

File: developer/openshift/dev_setup.sh [hunk 1]
```diff
@@ -168,6 +168,23 @@ cluster_setup() {
   | indent 2
 }
 
+# Sometimes CRDs need a few seconds to be ready
```
Comment (Roming22) [inline developer/openshift/dev_setup.sh:171]: I feel this should be moved to `developer/openshift/utils.sh`

**Review by Roming22 — CHANGES_REQUESTED** [review]

File: developer/openshift/dev_setup.sh (hunk 1 above)
Comment (xinredhat) [reply developer/openshift/dev_setup.sh]: fixed

**Pull request merged**